src/components/PatientProvider.tsx

```diff
@@ -1,27 +1,29 @@
-import React, { FC, createContext, useContext, useEffect, useState, ReactNode } from 'react';
-import { useFHIRClient } from './FHIRClient';
+import React, { FC, createContext, useContext, ReactNode } from 'react';
 import { Patient } from 'fhir-objects';
 interface PatientProviderProps {
   children: ReactNode;
-  patient?: Patient | null;
+  value: PatientContextInterface;
 }
 
-export const PatientContext = createContext<Patient | null>(null);
-
-export const PatientProvider: FC<PatientProviderProps> = ({ children, patient }) => {
-  const client = useFHIRClient();
-  const [currentPatient, setCurrentPatient] = useState<Patient | null>(patient || null);
+interface PatientContextInterface {
+  patient: Patient | null;
+  setPatient: Function;
```

Comment:
This was changed to follow the same pattern as `PatientRecordsProvider`. The `patient` state was moved into the `App` component so that it would know about changes to the patient. It seems like no other component currently needs the `setPatient` function so we could remove it but in the future if other components need to change the patient we would need to add this function back.